### PR TITLE
Use a tabular design for cross-section proposed page

### PR DESCRIPTION
**Pull request by dabreegster**
I'd want feedback internally before going with this approach.

Also sets up deploying experimental versions from branches, lifted from the atip repo, so we can share things like this without merging into main

### CODE REVIEW

File: src/routes/cross_section/proposed/CrossSection.svelte [hunk 1]
```diff
@@ -63,20 +90,142 @@
...
-      on:addRightBuffer={() => insertBuffer(i + 1)}
-      isFirst={i == 0}
-      isLast={i == streetFeaturesLeftToRight.length - 1}
-      leftFeature={i == 0 ? "" : streetFeaturesLeftToRight[i - 1]}
-      rightFeature={i == streetFeaturesLeftToRight.length - 1
-        ? ""
-        : streetFeaturesLeftToRight[i + 1]}
-    />
-  {/each}
+  <table>
```
Comment (dabreegster) [inline src/routes/cross_section/proposed/CrossSection.svelte:93]: Code-wise, it's unfortunate that HTML tables can't be defined in column-order, but it's not the worst thing

File: src/routes/cross_section/proposed/guidance.ts [hunk 2]
```diff
@@ -6,9 +6,9 @@ import type {
 import { calculateEffectiveSpeedLimit } from "./logic";
 
 export function references(
-  streetFeature: StreetFeatureType,
+  streetFeature: StreetFeatureType | "",
```
Comment (dabreegster) [inline src/routes/cross_section/proposed/guidance.ts:9]: It simplifies code to make these handle the unknown case

File: src/routes/cross_section/proposed/logic.ts [hunk 3]
```diff
@@ -271,11 +273,5 @@ export function calculateTotalWidths(
...
 export type BufferDetails = {
   warning: string;
-  functionsToDispatch: (
-    | "addRightBuffer"
-    | "addLeftBuffer"
-    | "delete"
-    | "moveLeft"
-    | "moveRight"
-  )[];
+  fix: "" | "left" | "right" | "both";
```
Comment (Pete-Y-CS) [inline src/routes/cross_section/proposed/logic.ts:276]: I like this, nicer for sure

**Review by Pete-Y-CS — APPROVED** [review]

**Comment by dabreegster** [comment]
We can always revert if the other style is preferred. But yeah, this feels like the way forward. Thanks for review

**Pull request merged**